DEVELOPERS.md

```diff
@@ -160,6 +160,21 @@ a = (b << 3) * 5;
  }
 ```
 
+* Structure in the C files
```

Comment:
yes i just moved them no single line changed in htere, but yeah, sucks that when such refactorings happen we can only do a blind assumption because there's no tool that diffs properly in those cases